### PR TITLE
sample packages

### PR DESCRIPTION
**Pull request by explizito**

### CODE REVIEW

**Comment by explizito** [comment]
The sample packages for now. Will reorganize later, but want to get this repo up first

**Review by iai-stephen — COMMENTED** [review]
Is the toplevel __init__.py needed? Can we also remove the __pychache__

**Comment by explizito** [comment]
> Is the toplevel **init**.py needed? Can we also remove the **pychache**

I think it may be needed for ticket where we remove import it for the e2e tests. I'll rm it for now, and add it again if need be

File: sample_notebook/integrateai_local_python_env_client_server.ipynb
Comment (shaydchara) [inline sample_notebook/integrateai_local_python_env_client_server.ipynb:1]: This example is an internal only one (for IAI I mean) so should not be pushed for public.

**Review by BrendanSchell — APPROVED** [review]

File: sample_notebook/integrateai_local_python_env_client_server.ipynb
Comment (explizito) [reply sample_notebook/integrateai_local_python_env_client_server.ipynb:1]: missed this, ty!

**Review by shaydchara — APPROVED** [review]

**Review by luwanwang — APPROVED** [review]
rubber stamp

**Review by SanjanaSridhar — APPROVED** [review]

**Review by SanjanaSridhar — APPROVED** [review]

**Pull request merged**